R/summary-log.R

```diff
@@ -218,7 +218,7 @@ extract_condition_number <- function(.s) {
   })
   .out <- map_dbl(.ofv, function(.x) {
     .cn <- .x[[SUMMARY_COND_NUM]]
-    if (is.null(.cn) || .cn == BBI_NULL_NUM) {
+    if (is.null(.cn) || .cn == BBI_NULL_NUM || is.na(.cn)) {
```

Comment:
Should `BBI_NULL_NUM` ever get this this point (i.e. is it still needed in this condition)?  Without looking closely at this, it seems to me that this could be reduced to an `is.null` check.

Reply:
Id still need the new NA call (evaluated to `logical(0)` without it), but I could likely get rid of the `BBI_NULL_NUM` part

Reply:
It had something to do with x being `NA`. If I type in `is.null(NA_real_)` I get `FALSE`, but I know I had to add that change because that function was evaluating to that after `BBI_NULL_NUM` was changed. Ill see if I can reproduce that case

Reply:
Made this change in latest commit (wasnt able to reproduce the issue, so not 100% sure why it works now)